lively.morphic/serialization.js

```diff
@@ -81,8 +82,27 @@ export async function saveWorldToResource (world = MorphicEnv.default().world, t
   } finally { i && i.remove(); }
 }
 
-export function copyMorph (morph) {
-  return deserializeMorph(serializeMorph(morph), { migrations, reinitializeIds: true });
+export function copyMorph (morph, realCopy = false) {
```

Comment:
So, `realCopy` is just the difference between copying a morph and copying its connections/comments?

Reply:
It is never possible to copy a morph with comments. Is is however still possible to grab a morph with comment, and grab utilized copy(). To differentiate these, actual copying uses realCopy = true